stackrox/base/services/kustomization.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - scanner-service.yaml
+  - scanner-db-service.yaml
+  - central-service.yaml
```

Add your common labels here:
```suggestion
  - central-service.yaml
commonLabels:
   app.kubernetes.io/component: scanner
    app.kubernetes.io/instance: stackrox-central-services
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: stackrox
    app.kubernetes.io/part-of: stackrox-central-services
    app.kubernetes.io/version: 3.70.0
    helm.sh/chart: stackrox-central-services-70.0.0
```
If you don't quite get this refer to [our deployment of prow](https://github.com/operate-first/apps/blob/master/prow/base/kustomization.yaml). This is a good practice because if we ever want to change this (for instance the `app.kubernetes.io/version`), we just have to change it in one place instead of every manifest.